src/core/lib/callback-handler.ts

```diff
@@ -216,6 +216,17 @@ export default async function callbackHandler(params: {
 
       return { session, user, isNewUser: true }
     }
+  } else if (account.type === "dbCredentials") {
```

Comment:
Does this need to be a separate `account.type` than `credentials`? The OAuth flows can seamlessly work with both `jwt` or `database`

Reply:
I didn't want it to disturb the current `Credential` flow, because credentials in `jwt` strategy doesn't need database at all. It's not problem for me to change this if any reviewer say that - like I say, it's just to not destroy whole internet 😄

Reply:
@KATT I removed recently added type `dbCredentials` to use existing one `credentials`